gps_data.csv

```diff
@@ -0,0 +1,10 @@
+-14.270999999999999,-170.132

```

Comment:
Can you add more comments about the file here

Reply:
Sorry, fixed in #4